.github/workflows/ci.yml

```diff
@@ -0,0 +1,39 @@
+name: CI
+
+on: [pull_request]
```

Comment:
https://github.com/libero/article-store/pull/74/checks?check_run_id=331381010 triggered so I'd leave it like this until it breaks.

Reply:
Looks like there's more granular options for creation etc: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#onevent_nametypes